Model/Sync/Catalog/Processor.php

```diff
@@ -242,6 +242,17 @@ public function syncItems($collectionItems, $storeId, $isVisibleVariantsSync = f
 
         $itemsToBeSyncedToYotpo = $items['sync_data'];
         foreach ($itemsToBeSyncedToYotpo as $itemEntityId => $yotpoFormatItemData) {
+            if ($this->coreConfig->syncResetInProgress($storeId, 'catalog')) {
+                $this->yotpoCatalogLogger->info(
+                    __(
+                        'Product sync is skipped because catalog sync reset is in progress
+                        - Magento Store ID: %1, Name: %2',
+                        $storeId,
+                        $this->coreConfig->getStoreName($storeId)
+                    )
+                );
+                continue;
+            }
```

Comment:
Why put it within `syncItem`'s for loop?
If we're skipping all of the orders of a certain store, this store shouldn't be taken into consideration in the first place in the current iteration.

Reply:
If we don't put it inside this loop, the cron will continue processing items in a batch without considering the sync reset activity.

Reply:
Comment here - https://github.com/YotpoLtd/magento2-module-core/pull/136#discussion_r851813587

Reply:
We will make these changes as required